web/src/recent_view_ui.js

```diff
@@ -871,21 +871,21 @@ function topic_offset_to_visible_area(topic_row) {
         return undefined;
     }
     const $scroll_container = $("#recent_view_table .table_fix_head");
```

Comment:
This is just a bonus commit and doesn't effect the bug fix. It just works slighly better around partially hidden rows.